modules/statics/src/map.ts

```diff
@@ -22,14 +20,18 @@ export class CoinMap {
    * @return {BaseCoin}
    */
   public get(key: string): Readonly<BaseCoin> {
-    if (this._map.has(key)) {
+    if (this.has(key)) {
       // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
       return this._map.get(key)!;
     }
 
     throw new CoinNotDefinedError(key);
   }
 
+  public has(key: string): boolean {
+    return this._map.has(key);
+  }
+
```

Comment:
This change could affect several things on WP. We should try a custom version on T02 and check if other coins have any issue...

Reply:
This is a new method, because it is new, nobody is using it.